packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -493,11 +493,17 @@ class Treemap extends StatelessWidget {
   Widget buildSubTreemaps() {
     return LayoutBuilder(
       builder: (context, constraints) {
-        return Stack(
-          children: buildTreemaps(
-            children: nodes,
-            width: constraints.maxWidth,
-            height: constraints.maxHeight,
+        if (constraints.maxHeight == 0 || constraints.maxWidth == 0) {
```

Comment:
what if constraints has a size of 1 x 1? can we guarantee that we don't overflow then?